### PR TITLE
main/libgudev: upgrade to 232

### PR DESCRIPTION
**Pull request by Cogitri**

### CODE REVIEW

File: main/libgudev/APKBUILD [hunk 1]
```diff
@@ -1,16 +1,13 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=libgudev
-pkgver=230
-pkgrel=2
+pkgver=232
+pkgrel=1
```
Comment (tcely) [inline main/libgudev/APKBUILD]:
```suggestion
pkgrel=0
```

**Comment by Cogitri** [comment]
Hey @andypost, could you look at this? I know @ncopa is the maintainer of this, but this an incredibly minor bump and the entirety of GNOME has been blocking for over a week on this now :c

**Comment by tcely** [comment]
> ```
> for file main/libgudev/APKBUILD:
> cd "$builddir" can be removed
> ```

We don't remove these lines in `main` to make backporting easier.

**Comment by Cogitri** [comment]
Ah, we don't have umockdev packaged right now, so I've disabled it - otherwise libgudev's tests fail. I guess I should add that as a comment?

**Comment by tcely** [comment]
A comment or mention in your commit message would be great!

**Comment by Cogitri** [comment]
Fixed.

**Comment by Cogitri** [comment]
Thanks

**Pull request closed without merging**